Add a folder in the main repo for conferences attended with VDK

dvalkova opened the pull request:
VDK conferences: Add a folder in the main repo to store the "Welcome pages" of the conferences we attend

Having a folder inside VDK repo gives visibility to the community on which conferences did we present. It also gives a direct link between the main VDK repo and the repo we use for storing VDK examples/workshops used at the conferences themselves (https://github.com/versatile-data-kit-demo). Also, people at the conference could directly star the VDK repo instead of the workshops repo.

Added a folder "events" with subfolders for AMLD and OpenFest at which we already presented/will present. Inside each conf. subfolder, there's a README which is the "home/welcome" page shown on the conference and any other files that might be relevant.

Cosmetic change/folder structure

tozka commented:
Let's name the folder with more genric name. I suggest "Events" because I can include the demo I did for the ML Community meetup as well - https://github.com/vdk-ml-community/vdk-demo

dvalkova commented:
> Let's name the folder with more genric name. I suggest "Events" because I can include the demo I did for the ML Community meetup as well - https://github.com/vdk-ml-community/vdk-demo

Folder renamed from "conferences" to "events".

doks5 commented:
What is the idea of the `backup` folders? Also, the READMEs' for both conferences point to the same "local/backup" instructions not the ones in their respective backup folders, which makes me think we can maybe move the backup folder up in the main `events` folder and remove the backup folders from the separate conferences folders?

dvalkova commented:
> What is the idea of the `backup` folders? Also, the READMEs' for both conferences point to the same "local/backup" instructions not the ones in their respective backup folders, which makes me think we can maybe move the backup folder up in the main `events` folder and remove the backup folders from the separate conferences folders?
> 
> Maybe also update the PR description to reflect the change in `conferences` folder name?

Agreed about the backup folders. A more appropriate place for them would be the https://github.com/versatile-data-kit-demo?tab=repositories account, so I will keep them only there, deleted them from this commit. 
I also updated the PR description.

doks5 commented on events/amld-workshop/README.md:
Where is this link supposed to point to?

dvalkova replied:
Just committed a change with updated link.

doks5 commented on events/openfest/README.md:
Ditto.

dvalkova replied:
Updated.
